pkg/infra/process.go

```diff
@@ -3,15 +3,23 @@ package infra
 import (
 	"fmt"
 	"os"
-	"strconv"
 	"time"
 
 	"github.com/pkg/errors"
 	log "github.com/sirupsen/logrus"
+	"gopkg.in/alecthomas/kingpin.v2"
 )
 
 const loglevel = "TAPE_LOGLEVEL"
 
+var (
+	app = kingpin.New("tape", "A tiny, handy tool to test performance of a deployed Fabric network")
+
+	run = app.Command("run", "Start the tape program").Default()
+	con = run.Flag("config", "Set the config file's path").Default("./config.yaml").Short('c').String()
```

Comment:
I am confusing with the `set` here. is that means I am able to `tape -c config.yaml` and then `tape run` or `./tape` directly?  if we have to`tape -c xxx -n xxxx` to start, so that we'd better remove set. use `config file` for short?

Reply:
The `config` and `number` flags are set default values. If no parameters for flags are specified in the startup command, the default values are used. If you want to specify a special value for `config` flag, you can directly use the `./tape run -c xxxxx` or `./tape run --config xxxxx` format to run the program.
We can use `./tape` or `./tape run` to run the program with the flags' default value, because `run` is set as the default command.

Reply:
1. not sure for default value...
1. again... I suppose here is not set....  but use.
what I understanding/meaning for `set`
``` bash
./tape -c /tmp/config.yaml
./tape -n 5000
./tape
# the tape will run with config file located at /tmp/config.yaml and 5000 txs.
```
I suppose tape will `use`
``` bash
./tape -c /tmp/config.yaml -n 5000
# the tape will run with config file located at /tmp/config.yaml and 5000 txs.
```

Reply:
as we don't set, so the description should be "use the config file at/on path"
	con = run.Flag("config", "Set the config file's path").Default("./config.yaml").Short('c').String()

Reply:
tx数量也许可以设置默认数值，但是config我认为还是显式得要求用户设置比较合适

Reply:
I removed the dafault value for the two flags. If there is a need in the future, we will increase it again.